### ISSUE TITLE
Image alt text (test content?)

### ISSUE THREAD

**Issue by claireristow** [Lullabot/lb-ebook-poc#40]
This issue can probably be closed right away since I think it's only a symptom of adding some quick test content. But the alt text should describe the image. It is currently the title of the ebook which isn't helpful for understanding the image.

Eg. https://ebooks.lullabot.com/ebooks/jponch-sample/chapters/chapter-1/ 

Current alt text: Why your CMS project will fail 
Better alt text: A cartoon of a person crawling out of a browser window surrounded by various icons including an image icon, a notification bell, and a checklist.

![Screenshot 2024-07-17 at 11 22 41 AM](https://github.com/user-attachments/assets/83f84946-882d-41c2-acc5-aa9eb348173a)

**Comment by sirkitree** [comment]
hrm, no it is relevant even to real content as we haven't really given thought to alt text, or how to enter that for an image. @huntermacd what do you think? possibly use front-matter for this, since the image is defined in front-matter already?
